### ISSUE TITLE
Ability to change font & button size, and add Repeat button (and maybe add the option to remove Album Art)

### ISSUE THREAD

**Issue by gebbet** [devinbergin/spot-thing#140]
If possible, please add the ability to change the font & button size to make it smaller and maybe add the repeat (all and once) button from Spotify too? The fact that the font and the buttons are so big that the song title cannot be read when the spot-thing window is resized to a smaller size kinda defeats the purpose of the "mini-player" vibe. Also, maybe add the option to remove the album art as well? 

Basically, I want to reduce the total size of the app to something like the size of the bottom button bar but currently it is not possible. But it is still a great small app that suits my needs, so thanks a lot! :)

**Comment by devinbergin** [comment]
Thanks for the feedback! Glad you're enjoying spot-thing <3

I finally got around to wrapping up some pending changes I had to add the option to like a song that's playing so I added in the repeat functionality you requested as well. You can try it out by downloading the latest release.

Thanks again!
